src/components/Users/Users.vue

```diff
@@ -0,0 +1,36 @@
+<template src="./Users.html"></template>
+<style scoped src="./Users.css"></style>
+
+<script>
+  import User from './User/User';
+  import Scrollable from '@/components/Scrollable/Scrollable';
+  import { userService } from '@/services/userService';
+
+  export default {
+    name: 'Users',
+    components: {
+      User,
+      Scrollable,
+    },
+    data() {
+      return {
+        filter: {
+          gender: null
+        },
+        users: [],
+      };
+    },
+    watch: {
```

Comment:
Good solution)
But on a large form it can be complicated operation, so I prefer to use third party library like vuelidate (especially when we talking about real project)